incubator/jaeger/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.2.0
 description: A Jaeger Helm chart for Kubernetes
 name: jaeger
-version: 0.3.13
+version: 0.3.14
```

Comment:
This adds new features, so please bump the minor.

Reply:
ok, done. thanks!